src/IO/ReadHelpers.h

```diff
@@ -577,13 +636,18 @@ inline void readUUIDText(UUID & uuid, ReadBuffer & buf)
     char s[36];
     size_t size = buf.read(s, 36);
```

Comment:
But how we will continue to read next data from buffer if we already read extra bytes?